### PR TITLE
Automate Interaction Container Loading & Rename Missions

### PR DESCRIPTION
**Pull request by zach2good**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to Topaz Next's [Limited Contributor License Agreement](https://github.com/DerpyProjectGroup/topaz/blob/info/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/DerpyProjectGroup/topaz/blob/info/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: scripts/globals/interaction/interaction_global.lua [hunk 1]
```diff
@@ -32,7 +32,7 @@ function InteractionGlobal.loadContainers(shouldReloadRequires)
         package.loaded[interactionContainersPath] = nil
     end
 
-    local containerFiles = require(interactionContainersPath)
+    local containerFiles = GetQuestAndMissionFilenamesList()
```
Comment (claywar) [inline scripts/globals/interaction/interaction_global.lua:43]: From the CI fail, since a list is being returned, shouldn't the require still be necessary?

Comment (zach2good) [reply scripts/globals/interaction/interaction_global.lua:43]: After some digging, the CI fails were due to some other bits and pieces.

**Review by claywar — APPROVED** [review]

**Comment by TeoTwawki** [comment]
needs rebased

**Comment by TeoTwawki** [comment]
that works too I guess

**Pull request merged**